src/Bubble.js

```diff
@@ -52,6 +53,20 @@ export default class Bubble extends React.Component {
     return null;
   }
 
+  renderTicks() {
+    _renderTick = function(displayTick) {
```

Comment:
Creating nested methods is very wasteful and not necasery in this case, so I think `_renderTick` should be a method of Bubble, or even as a stateless component.
Also looks like it is a global variable - did you forget the `const`?